flux-driver/src/collector.rs

```diff
@@ -591,6 +655,13 @@ mod errors {
         pub span: Span,
     }
 
+    #[derive(SessionDiagnostic)]
+    #[error(code = "FLUX", slug = "parse-invalid-constant")]
```

Comment:
You have to add the error to this file https://github.com/liquid-rust/flux/blob/main/flux-errors/locales/en-US/parse.ftl